From 3cac7eeb9b7de43cd987de0b19c51fefb8ee3e3a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:07 -0500
Subject: [redhat] kabi: Adding symbol __bitmap_parse (lib/bitmap.c)

Message-id: <2b8cd1de4d7765177c674b668aeeef6eb0487327.1416318767.git.poros@redhat.com>
Patchwork-id: 100126
O-Subject: [PATCH RHEL7.1 02/63] kabi: Adding symbol __bitmap_parse (lib/bitmap.c)
Bugzilla: 1070274
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Used by the bitmap_parse() macro.
    Needed for cpumasks (for tying resources to cpus).
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 04/80] kabi: Adding symbol __bitmap_parse (lib/bitmap.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:04 +0100)
 Description:
 ============
 Used by the bitmap_parse() macro.
 Needed for cpumasks (for tying resources to cpus).

 Reviews:
 ========
 Re: [PATCH RHEL7.0 04/80] kabi: Adding symbol __bitmap_parse (lib/bitmap.c)
 Prarit Bhargava <prarit@redhat.com> (Wed, 05 Mar 2014 07:52:41 -0500)
 Acked-by: Prarit Bhargava <prarit@redhat.com>
 P.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__bitmap_parse
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__bitmap_parse
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__bitmap_parse

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__bitmap_parse b/redhat/kabi/kabi-module/kabi_ppc64/__bitmap_parse
new file mode 100644
index 0000000..f99a110
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__bitmap_parse
@@ -0,0 +1,2 @@
+#1-
+0x4df119fa __bitmap_parse vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__bitmap_parse b/redhat/kabi/kabi-module/kabi_s390x/__bitmap_parse
new file mode 100644
index 0000000..f99a110
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__bitmap_parse
@@ -0,0 +1,2 @@
+#1-
+0x4df119fa __bitmap_parse vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__bitmap_parse b/redhat/kabi/kabi-module/kabi_x86_64/__bitmap_parse
new file mode 100644
index 0000000..f99a110
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__bitmap_parse
@@ -0,0 +1,2 @@
+#1-
+0x4df119fa __bitmap_parse vmlinux EXPORT_SYMBOL
-- 
1.7.1